File: 2022/20221129/MaxAndMin.test.js
import solution from './MaxAndMin';

test('MaxAndMin', () => {
  expect(solution('1 2 3 4')).toBe('1 4');
  expect(solution('-1 -2 -3 -4')).toBe('-4 -1');
  expect(solution('-1 -1')).toBe('-1 -1');
});

File: 2022/20221128/MaxAndMin/src/test/java/MaxAndMinTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class MaxAndMinTest {
    @Test
    void MaxAndMin() {
        MaxAndMin test = new MaxAndMin();

        assertEquals(test.solution("1 2 3 4"), "1 4");
        assertEquals(test.solution("-1 -2 -3 -4"), "-4 -1");
        assertEquals(test.solution("-1 -1"), "-1 -1");
    }
}
